ext/LinearSolvePardisoExt.jl

```diff
@@ -132,6 +132,11 @@ function SciMLBase.solve!(cache::LinearSolve.LinearCache, alg::PardisoJL; kwargs
     @unpack A, b, u = cache
     A = convert(AbstractMatrix, A)
     if cache.isfresh
+        if hasproperty(alg, :precs) && !isnothing(alg.precs)
+            Pl, Pr = cache.alg.precs(x, cache.p)
+            cache.Pl = Pl
+            cache.Pr = Pr
+        end
```

Comment:
Why is this needed here ? Pl, Pr are not used by Pardiso.

Reply:
oh, is krylov the only one that uses them? I thought there was another...

Reply:
IterativeSolvers, KrylovKit.